### PR TITLE
buffed docs for awx release and canonical triage responses

### PR DESCRIPTION
**Pull request by sarabrajsingh**
##### ISSUE TYPE

 - Bug or Docs Fix

##### COMPONENT NAME

 - Docs

### CODE REVIEW

**Comment by sarabrajsingh** [comment]
looks like my linter picked up some changes but this should be fine @john-westcott-iv

File: docs/release_process.md [hunk 1]
```diff
@@ -135,3 +136,25 @@ We're happy to announce that [AWX version 21.1.0](https://github.com/ansible/awx
...
+
+![Revert-2-Image](img/revert-2.png)
+
+3. Navigate to [quay.io](https://quay.io/repository/ansible/awx?tag=latest&tab=tags) and delete the published AWX image(s) and tags.
+
+4. Navigate to [quay.io](https://github.com/ansible/awx-operator/releases) and delete the published AWX Operator image(s) and tags.
+
+5. Navigate to the [Ansible Galaxy Collections](https://galaxy.ansible.com/awx/awx) website and remove the published AWX collection with the bad tag.
+
+6. Navigate to the [PyPi](https://pypi.org/project/awxkit/#history) and delete the bad AWX tag and release that got published.
```
Comment (john-westcott-iv) [inline docs/release_process.md]: Did we confirm that we can actually do this? If not maybe we just need to say something like "If items were released we need to just do another z stream release".

File: docs/release_process.md [hunk 2]
```diff
@@ -135,3 +136,25 @@ We're happy to announce that [AWX version 21.1.0](https://github.com/ansible/awx
...
 
 ## Send the same IRC message (less the @newsbot) to #awx:ansible.com
+
+## Revert a Release
+
+Here are the steps needed to revert an AWX and an AWX-Operator release. Depending on your use case, follow the steps for reverting just an AWX release, an Operator release or both.
+
+1. Navigate to the [AWX Release Page](https://github.com/ansible/awx/releases) and delete the AWX Release that needs to be removed.
+
+![Revert-1-Image](img/revert-1.png)
```
Comment (john-westcott-iv) [inline docs/release_process.md:149]: Didn't we also need to delete the tag?

File: .github/triage_replies.md [hunk 3]
```diff
@@ -79,31 +79,31 @@ The Ansible Community is looking at building an EE that corresponds to all of th
...
-We're happy to announce that the next release of AWX, version <X> is now available! \
-In addition AWX Operator version <Y> has also been release! \
+We're happy to announce that the next release of AWX, version <b>`Xa.Ya.za`</b> is now available! \
+In addition AWX Operator version <b>`Xb.Yb.zb`</b> has also been released! \
 \
 Please see the releases pages for more details: \
-	AWX: https://github.com/ansible/awx/releases/tag/<X> \
-	Operator: https://github.com/ansible/awx-operator/releases/tag/<Y> \
+	AWX: https://github.com/ansible/awx/releases/tag/Xa.Ya.za \
+	Operator: https://github.com/ansible/awx-operator/releases/tag/Xb.Yb.zb \
```
Comment (john-westcott-iv) [inline .github/triage_replies.md:106]: Just a heads up, @gundalow would like us to add some high level release notes. I modified my version of the get_release_notes script to provide a bulleted list of any [XY] level changes. I'll update the docs for this after you merge these changes.

File: docs/release_process.md (hunk 1 above)
Comment (sarabrajsingh) [reply docs/release_process.md]: we can certainly mention both methods (this cleanup and a fall-forward method).

File: docs/release_process.md [hunk 4]
```diff
@@ -135,3 +136,33 @@ We're happy to announce that [AWX version 21.1.0](https://github.com/ansible/awx
 We're happy to announce that [AWX Operator version 0.22.0](https://github.com/ansible/awx-operator/releases/tag/0.22.0) is now available!
 
 ## Send the same IRC message (less the @newsbot) to #awx:ansible.com
+
+## Revert a Release
+
```
Comment (gundalow) [inline docs/release_process.md:141]: Should we give examples of when we might want/not to revert a release?
ie when to revert, and when to make a new release instead?

**Review by gundalow — COMMENTED** [review]
Nice work :)

**Review by john-westcott-iv — APPROVED** [review]

File: docs/release_process.md (hunk 4 above)
Comment (sarabrajsingh) [reply docs/release_process.md:141]: i just have a blurb saying "if you need to revert" - i guess it's up to the developer and/or awx team to give the thumbs up/down on a revert. what do you think @gundalow ?

**Comment by sarabrajsingh** [comment]
@john-westcott-iv i couldn't use my fork to mock an awx deploy and remove the tags-created through the ui because of secrets permissions issues. could we take screenshots during the next official AWX release and add them to the docs here? we can use real AWX to pseudo-deploy, backout & take screenshots, and then continue onwards with the release. what do you think?

**Comment by john-westcott-iv** [comment]
That sounds fine, do you want to add "placeholder" images or anything in this PR?

**Comment by sarabrajsingh** [comment]
@john-westcott-iv - i left a comment in markdown here (https://github.com/ansible/awx/pull/12509/files#diff-b54fc682b6f5e7b977b36d98dd902b2bbcb5aef2a75b306f493dc0768723b33cR154) in the file

**Pull request merged**